db/migrations/0002_actor.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.0.2 on 2023-12-31 19:38
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Actor',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_name', models.CharField(max_length=255)),
+                ('last_name', models.CharField(max_length=255)),
+            ],
+        ),
+    ]
```

Comment:
It is a good practice to include one migration per app per pull request in a Django project that includes all changes made to models within this pull request. It helps to reduce the `migrations` folder overloading in the future. Please, include one migration in your PR.